packages/block-library/src/verse/style.scss

```diff
@@ -1,5 +1,4 @@
 pre.wp-block-verse {
 	font-family: inherit;
 	overflow: auto;
-	white-space: nowrap;
```

Comment:
In the original PR, [a `pre-wrap` was considered here](https://github.com/WordPress/gutenberg/pull/27734#issuecomment-745292725). Should we use it explicitly instead of removing the whole thing? AFAIK the `pre` tag's default `white-space` value is `pre`. cc @scruffian @kjellr

Reply:
It looks like the richtext editor forces a wrap regardless of what we do here: 

<img width="881" alt="Screen Shot 2021-01-11 at 2 39 29 PM" src="https://user-images.githubusercontent.com/1202812/104229873-d1c28080-541a-11eb-8a5b-09ef3baeef33.png">

Currently in the this PR, removing the rule (and defaulting to `whitespace: pre`) keeps long words in the same line, and scrolls them horizontally if necessary: 

<img width="607" alt="Screen Shot 2021-01-11 at 2 43 38 PM" src="https://user-images.githubusercontent.com/1202812/104230314-675e1000-541b-11eb-9107-96c732c61db6.png">

If we did go with `pre-wrap`, very long words _would_ wrap, though not exactly the same way they do in the editor. In my testing, they wrap onto their own single line, but then don't continue wrapping beyond there, instead forcing a horizontal scroll: 

<img width="611" alt="Screen Shot 2021-01-11 at 2 43 32 PM" src="https://user-images.githubusercontent.com/1202812/104230331-6cbb5a80-541b-11eb-8471-d3eeb98a1a77.png">

---

So given that, I'd recommend `pre-wrap` since it's closer to what the editor does (but I'd also suggest that the better solution would be to try to get the editor to mimic `whitespace: pre` and stop wrapping the text).

Reply:
> So given that, I'd recommend pre-wrap since it's closer to what the editor does in the near term

@kjellr Cool. I'm happy to help with that 👍